openmls/src/ciphersuite/mod.rs

```diff
@@ -104,21 +104,26 @@ impl Ciphersuite {
             return Err(ConfigError::UnsupportedCiphersuite);
         }
 
-        Ok(Ciphersuite {
+        Ok(Self::new_from_supported(name))
+    }
+
+    /// This creates a Ciphersuite from one of the supported ciphersuite names.
+    /// This should only be used if it is clear the ciphersuite is used.
```

Comment:
I'm having trouble understanding the second line. Should it be "... the ciphersuite is supported."?

Reply:
I see. But shouldn't this throw an error then, if the ciphersuite is not supported? Which makes me wonder what currently happens if this is called with an unsupported ciphersuite.

Taking a closer look, the issue of ciphersuite support seems a bit inconsistent. There are two functions `supported_ciphersuites`, which accesses the `Config` and `supported_ciphersuite_names`, which accesses a constant that is independent of the config.

So either way, I think this function should at least get a comment saying what happens if the input ciphersuite is not supported. Regarding the general inconsistency of the config, I'll file a follow-up.